include/nil/crypto3/zk/commitments/polynomial/kzg.hpp

```diff
@@ -322,20 +318,27 @@ namespace nil {
                         bool>::type = true>
                 static void update_transcript(const typename KZG::public_key_type &public_key,
                                             typename KZG::transcript_type &transcript) {
-                    std::vector<std::uint8_t> byteblob(KZG::scalar_blob_size);
+                    nil::marshalling::status_type status;
 
                     for (const auto &commit : public_key.commits) {
-                        transcript(KZG::serializer::point_to_octets(commit));
+                        std::vector<uint8_t> byteblob =
```

Comment:
I can see that whatever is in public_key goes into transcript. This means you need to create a structure in marshalling for public_key, and marshall it, then move a single "std::vector<uint8_t> byteblob" into transcript, not marshall each part separately.

Reply:
Just add a comment here, saying this is temporary. Put a link to a task/issue in the comment, and that's it.